lib/test.js

```diff
@@ -11,6 +11,7 @@ var inspect = require('util').inspect;
 var assert = require('./assert');
 var enhanceAssert = require('./enhance-assert');
 var globals = require('./globals');
+var plur = require('plur');
```

Comment:
Should be above relative require calls. Wish we had a ESLint rule for this. (https://github.com/sindresorhus/module-requests/issues/62)